JM/COT/languagecore/stringtable.csv

```diff
@@ -1,181 +1,181 @@
+STR_COT_ESP_MODULE_VIEW_TYPE_TRANSMITTERS,Transmitters,Transmitters,Transmitters,Transmitters,Передатчики,Transmitters,Transmitters,Transmitters,Transmitters,Transmetteurs,Transmitters,Transmitters,Transmitters,Transmitters
+STR_COT_ESP_MODULE_VIEW_TYPE_UNKNOWN_ITEMS,Unknown Items,Unknown Items,Unknown Items,Unknown Items,Неизвестные предметы,Unknown Items,Unknown Items,Unknown Items,Unknown Items,Objets inconnus,Unknown Items,Unknown Items,Unknown Items,Unknown Items
+STR_COT_ESP_MODULE_VIEW_TYPE_WEAPONS,Weapons,Weapons,Weapons,Weapons,Оружие,Weapons,Weapons,Weapons,Weapons,Armes,Weapons,Weapons,Weapons,Weapons
+STR_COT_FORMAT_METRE,%1m,%1m,%1m,%1m,%1м,%1m,%1m,%1m,%1m,%1m,%1m,%1m,%1m,%1m
+STR_COT_FORMAT_METRE_LONG,%1 metre(s),%1 metre(s),%1 metre(s),%1 metre(s),%1 метр(-ов),%1 metre(s),%1 metre(s),%1 metre(s),%1 metre(s),%1 metre(s),%1 metre(s),%1 metre(s),%1 metre(s),%1 metre(s)
 STR_COT_FORMAT_NONE,%1,%1,%1,%1,%1,%1,%1,%1,%1,%1,%1,%1,%1,%1
 STR_COT_FORMAT_PERCENTAGE,%1%,%1%,%1%,%1%,%1%,%1%,%1%,%1%,%1%,%1%,%1%,%1%,%1%,%1%
 STR_COT_FORMAT_SECOND,%1s,%1s,%1s,%1s,%1s,%1s,%1s,%1s,%1s,%1s,%1s,%1s,%1s,%1s
-STR_COT_FORMAT_SECOND_LONG,%1 second(s),%1 second(s),%1 second(s),%1 second(s),%1 second(s),%1 second(s),%1 second(s),%1 second(s),%1 second(s),%1 seconde(s),%1 second(s),%1 second(s),%1 second(s),%1 second(s)
-STR_COT_GENERIC_CANCEL,Cancel,Cancel,Cancel,Cancel,Cancel,Cancel,Cancel,Cancel,Cancel,Annuler,Cancel,Cancel,Cancel,Cancel
-STR_COT_GENERIC_CREATE,Create,Create,Create,Create,Create,Create,Create,Create,Create,Créer,Create,Create,Create,Create
-STR_COT_GENERIC_MAKE,Make,Make,Make,Make,Make,Make,Make,Make,Make,Faire,Make,Make,Make,Make
-STR_COT_GENERIC_NO,No,No,No,No,No,No,No,No,No,Non,No,No,No,No
-STR_COT_GENERIC_OFF,off,off,VYP,AUS,ВЫКЛ,WYŁ.,off,NO,NO,NON,關,オフ,off,关
+STR_COT_FORMAT_SECOND_LONG,%1 second(s),%1 second(s),%1 second(s),%1 second(s),%1 сек.,%1 second(s),%1 second(s),%1 second(s),%1 second(s),%1 seconde(s),%1 second(s),%1 second(s),%1 second(s),%1 second(s)
+STR_COT_GENERIC_CANCEL,Cancel,Cancel,Cancel,Cancel,Отменить,Cancel,Cancel,Cancel,Cancel,Annuler,Cancel,Cancel,Cancel,Cancel
+STR_COT_GENERIC_CREATE,Create,Create,Create,Create,Создать,Create,Create,Create,Create,Créer,Create,Create,Create,Create
+STR_COT_GENERIC_MAKE,Make,Make,Make,Make,Создать,Make,Make,Make,Make,Faire,Make,Make,Make,Make
+STR_COT_GENERIC_NO,No,No,No,No,Нет,No,No,No,No,Non,No,No,No,No
+STR_COT_GENERIC_OFF,off,off,VYP,AUS,выкл,WYŁ.,off,NO,NO,NON,關,オフ,off,关
 STR_COT_GENERIC_OFF_C,OFF,OFF,VYP,AUS,ВЫКЛ,WYŁ.,OFF,NO,NO,NON,關,オフ,OFF,关
-STR_COT_GENERIC_ON,on,on,ZAP,AN,ВКЛ,WŁ.,on,SÌ,SÍ,OUI,開,オン,on,开
+STR_COT_GENERIC_ON,on,on,ZAP,AN,вкл,WŁ.,on,SÌ,SÍ,OUI,開,オン,on,开
 STR_COT_GENERIC_ON_C,ON,ON,ZAP,AN,ВКЛ,WŁ.,ON,SÌ,SÍ,OUI,開,オン,ON,开
 STR_COT_GENERIC_SPAWN,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn,Spawn
 STR_COT_GENERIC_YES,Yes,Yes,Yes,Yes,Yes,Yes,Yes,Yes,Yes,Oui,Yes,Yes,Yes,Yes
```

Comment:
Missing translation

Reply:
I can't find what I missed here, everything seems ok to me

Reply:
STR_COT_GENERIC_YES is still "Yes" for russian